src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -34,11 +33,34 @@ const ApplicationModal = ({
   const handleCancel = () => {
     setDisplayModal(false);
     setProfileId('');
-    setHideForm(true);
   };
 
-  const displayForm = () => {
-    setHideForm(false);
+  const openNotificationWithIcon = (type, status, err) => {
+    if (type === 'success') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User has been approved successfully',
+        });
+      } else {
+        notification[type]({
+          message: 'User has been rejected successfully',
+        });
+      }
+    }
+
+    if (type === 'error') {
+      if (status === 'approve') {
+        notification[type]({
+          message: 'User could not be approved at this time',
+          description: `Error: ${err}`,
+        });
+      } else {
+        notification[type]({
+          message: 'User could not be rejected at this time',
+          description: `Error: ${err}`,
```

Comment:
Nice touch here too - I always appreciate giving the user some feedback to let them know something went wrong and why!